src/pages/commits/commitChart/Grid.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { uiColors } from "@leafygreen-ui/palette";
+
+const { gray } = uiColors;
+
+export const Grid: React.FC<{
+  numDashedLine: number;
+}> = ({ numDashedLine }) => (
+  <ColumnContainer>
+    {[...Array(numDashedLine)].map(() => (
+      <DashedLine />
```

Comment:
DashedLine needs a unique `key` prop. You can see a warning about it the console when we render this component